packages/arb-token-bridge-ui/src/util/wagmi/setup.ts

```diff
@@ -26,29 +27,104 @@ const chainList = isTestingEnvironment
     ]
   : [mainnet, arbitrum, arbitrumNova, goerli, arbitrumGoerli]
 
-const { chains, provider } = configureChains(chainList, [publicProvider()])
+const projectId = process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID!
```

Comment:
NIT: can we check if it exists? Just in case

Reply:
🤷 If it doesn't the app will break anyway